integration/tests/cook/test_basic.py

```diff
@@ -209,6 +209,29 @@ def test_mea_culpa_retries(self):
         finally:
             util.kill_jobs(self.cook_url, [uuid])
 
+    def test_compute_cluster(self):
+        settings_dict = util.settings(self.cook_url)
+
+        # For now, we only have one compute cluster. This could be wrong with future
+        # refactors, but for now, this is the only compute cluster we have, so we should
+        # expect it.
+        expected_compute_cluster = settings_dict['mesos-compute-cluster-name']
+        self.assertIsNotNone(expected_compute_cluster)
+        self.assertEqual("default-compute-cluster-from-config-defaulting",expected_compute_cluster)
+
+        job_executor_type = util.get_job_executor_type(self.cook_url)
+        job_uuid, resp = util.submit_job(self.cook_url, executor=job_executor_type)
+        self.assertEqual(resp.status_code, 201, msg=resp.content)
+        self.assertEqual(resp.content, str.encode(f"submitted jobs {job_uuid}"))
```

Comment:
You can remove this assertion from this test, it's covered elsewhere I believe.

Reply:
Ok.